scylla-cql/src/frame/value.rs

```diff
@@ -26,6 +20,18 @@ pub struct ValueTooBig;
 #[error("Value is too large to fit in the CQL type")]
 pub struct ValueOverflow;
 
+#[derive(Debug, Error, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub enum SerializeValuesError {
+    #[error("Too many values to add, max 65,535 values can be sent in a request")]
+    TooManyValues,
+    #[error("Mixing named and not named values is not allowed")]
+    MixingNamedAndNotNamedValues,
+    #[error(transparent)]
+    ValueTooBig(#[from] ValueTooBig),
+    #[error("Parsing serialized values failed")]
+    ParseError,
+}
+
```

Comment:
Why is this not part of the legacy module?

Reply:
ditto for `ValueTooBig`

Reply:
`SerializeValuesError` is used in `types::serialize::row::SerializedValues::add_value`, and `SerializeValuesError` contains a variant for `ValueTooBig`. That's why I left those not deprecated.